src/ProductConstructionService/ProductConstructionService.FeedCleaner/FeedCleaner.cs

```diff
@@ -31,6 +31,42 @@ public FeedCleaner(
         _httpClient = httpClientFactory.CreateClient();
     }
 
+    /// <summary>
+    /// Symbol feeds are not cleaned package by package bu rather removed once the matching non-symbol feed is deleted.
+    /// </summary>
+    public async Task CleanSymbolFeedAsync(List<AzureDevOpsFeed> allFeeds, AzureDevOpsFeed symbolFeed)
```

Comment:
```suggestion
    public async Task CleanSymbolFeedAsync(List<AzureDevOpsFeed> packageFeeds, AzureDevOpsFeed symbolFeed)
```
maybe rename this since we're passing only them?